instrumentation/elasticsearch/elasticsearch-transport-5.3/javaagent/elasticsearch-transport-5.3-javaagent.gradle

```diff
@@ -5,19 +5,17 @@ muzzle {
     group = "org.elasticsearch.client"
     module = "transport"
     versions = "[5.3.0,6.0.0)"
-    // TODO (trask) try to remove the skipVersions in the future
-    //  not sure what's wrong with this recent version, hopefully just publishing blip that
-    //  will work itself out
+    // version 7.11.0 depends on org.elasticsearch:elasticsearch:7.11.0 which depends on
+    // org.elasticsearch:elasticsearch-plugin-classloader:7.11.0 which does not exist
     skipVersions += ['7.11.0']
```

Comment:
If this is needed here, why it's not needed in `elasticsearch-transport-5.0-javaagent.gradle` above?

Reply:
this is a good question, may be getting lucky, I just sent a temporary commit to remove the muzzle range limit so it will test everything

Reply:
Good call 👍, I will fix the remaining failures (4!) tomorrow.